src/main.zig

```diff
@@ -1179,6 +1179,16 @@ fn buildOutputType(
                     .wl => {
                         var split_it = mem.split(it.only_arg, ",");
                         while (split_it.next()) |linker_arg| {
+                            // Handle nested-joined args like `-Wl,-rpath=foo`.
+                            // Must be prefixed with 1 or 2 dashes.
+                            if (linker_arg.len >= 3 and linker_arg[0] == '-' and linker_arg[2] != '-') {
+                                if (mem.indexOfScalar(u8, linker_arg, '=')) |equals_pos| {
+                                    try linker_args.append(linker_arg[0..equals_pos]);
+                                    try linker_args.append(linker_arg[equals_pos + 1 ..]);
+                                    continue;
+                                }
+                            }
```

Comment:
Are you sure this is correct? Seems weird that it's asking to pass `-rpath=foo` to the linker, but instead we pass `-rpath foo`

Reply:
This is correct. Both macos and linux system linkers accepts BOTH 2-arg and 1-arg-with-equals syntax. Ultimately, we choose the easiest way for our linker-args pass to just see it as 2-arg (which it already supported before this PR).